.github/workflows/build.yml

```diff
@@ -4,6 +4,8 @@ on:
   workflow_dispatch:
   push:
   pull_request:
+    branches-ignore:    
+      - 'dependabot/*'
```

Comment:
Great. I'm just wondering if we should do it the other way round. Instead of excluding dependabot pull requests, we can exclude dependabot branch pushes. That would mean that another contributor can still open a pull request with a branch name like `dependabot/serious-upgrade` and it will be build only once on our repository. That would look like this:

```yaml
  push:
    branches-ignore:    
      - 'dependabot/*'
  pull_request:
```

That raises another issue for me: When I push a branch to my fork, it gets build there which can be very handy to get early feedback before opening a pull request. But when I open a pull request, it gets build again for the main repository. All subsequent pushes get build twice, here and on my fork. I wish there was an option to ignore branch builds when a pull request exists. We could check that through some git fetches, I think. But it gets complicated. So this solution is the best I can think of for now.